resources/resources/software/aws/toolkits/resources/localized_messages.properties

```diff
@@ -176,6 +176,7 @@ s3.download.object.action=Download...
 s3.download.object.failed=Failed to download object
 s3.download.object.description=Download
 s3.download.object.progress=Downloading ''{0}''
+s3.load_more=Load More...
```

Comment:
nit: should this be title case? feel like it should be `Load more...` or even `load more...`

Reply:
Agreed, it probably should not be title case